src/frontend/src/service/api-service.test.ts

```diff
@@ -1,3 +1,4 @@
+//NOTE, JSON data has changed, this file need to be updated
```

Comment:
Needs a space before `NOTE` -- https://github.com/airbnb/javascript#comments--spaces

This comment itself is really unclear. By "JSON data" do you mean the format of the mock record objects?

Reply:
Space has been added and comment is now more clear.  Yes, the response from the API no longer looks like what the test implies.